data/wesionary-team.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "wesionaryTEAM Co. Ltd.",
+    "establishedAt": 1580667300,
+    "location": "KVD Complex 2F, Balkumari - 08, Lalitpur",
+    "website": "https://en.wesionary.team/",
+    "email":"hr@wesionary.team",
+    "phone":"",
+    "socials": {
+      "facebook": "https://www.facebook.com/wesionaryteam",
+      "instagram": null,
```

Comment:
This may be the insta link for the company? https://www.instagram.com/wesionaryteam/ 👀

Reply:
Looks to be the case to me as well